util/Migrator/DbScripts/2024-09-13_00_VerifiedOrgDomainSsoDetails.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR ALTER PROCEDURE [dbo].[OrganizationDomainSsoDetails_ReadByEmail]
```

Comment:
In addition to this migration you will want to update the [declaration](https://github.com/bitwarden/server/blob/f72932bf2496e70776a5fb81f0d4734b3807da9d/src/Sql/dbo/Stored%20Procedures/OrganizationDomainSsoDetails_ReadByEmail.sql#L4) of this stored procedure in the `SQL` project.